src/app.js

```diff
@@ -6,23 +6,23 @@ export function app() {
     const response = await fetch('https://pokeapi.co/api/v2/pokemon?limit=5');
     const results = await response.json();
     const pokemons = await results.results;
+
     const pokemonData = [];
 
-    pokemons.forEach(async pokemon => {
-      const response = await fetchPokemon(pokemon);
-      pokemonData.push(response);
-      const pokemonDiv = document.createElement('div');
-      pokemonDiv.innerText = pokemonData.name;
+    pokemons.forEach(pokemon => {
+      // Sorry for the dirty implementation of pictures.
+
+      const subStrings = pokemon.url.split('/');
+      const pokemonId = subStrings[6];
+      const imagePath =
+        'https://pokeres.bastionbot.org/images/pokemon/' + pokemonId + '.png';
```

Comment:
Looks like you copied all pokemon images to this project. 
Fair enough, but you not using them right now?

Reply:
Yes. But I got stuck using them and opened https://github.com/neuefische/cgn/issues/14

I will resolve this in another branch only related to file hosting.